examples/stm32h7/memory.x

```diff
@@ -0,0 +1,5 @@
+MEMORY
```

Comment:
this memory.x is unused, the build is using the  one generated by embassy-stm32 because  it has the `memory-x` feature